config/default/core.entity_form_display.user.user.register.yml

```diff
@@ -0,0 +1,38 @@
+uuid: 094cc3cc-6348-4987-953e-50088c3c2267
```

Comment:
@ab-ew what is the reason for these changes here?
Everything else looks fine, so feel free to merge if it is expected change